forgit.plugin.zsh

```diff
@@ -54,12 +54,16 @@ forgit::diff() {
         fi
     }
     repo="$(git rev-parse --show-toplevel)"
-    cmd="cd '$repo' && echo {} |sed 's/.*] *//' | sed 's/  ->  / /' |xargs -I% git diff --color=always $commits -- % | $forgit_diff_pager"
+    get_files="cd '$repo' && echo {} | sed 's/.*] *//' | sed 's/  ->  / /'"
+    preview_cmd="$get_files | xargs -I% git diff --color=always $commits -- % | $forgit_diff_pager"
+    # Show additional context on enter compared to preview
+    enter_cmd="$get_files | xargs -I% git diff --color=always -U10000 $commits -- % | $forgit_diff_pager"
```

Comment:
Maybe `10` is enough? Imagine there will be too many irrelevent lines displied when you modified a large file.

Reply:
> Would you accept making the diff context size configurable via a variable?

LGTM!

Reply:
@wfxr Great, I added a commit which introduces variables for the diff context. Can you check?